build.gradle

```diff
@@ -210,3 +231,7 @@ task updateVersion {
         ant.replaceregexp(file:'build.gradle', match: '"opensearch.version", "\\d.*"', replace: '"opensearch.version", "' + newVersion.tokenize('-')[0] + '-SNAPSHOT"', flags:'g', byline:true)
     }
 }
+
+configurations.all {
```

Comment:
Where jackson is leaking?

Reply:
Was getting conflicting version numbers for jackson with `./gradlew assemble` but not any more (not sure why). Will remove.